charts/ingress-nginx/values.yaml

```
@@ -754,6 +754,8 @@ controller:
     labels: {}
     # -- Use an existing PSP instead of creating one
     existingPsp: ""
+    rbac:
+      create: true
```

I know, there also are no docs for the other `create` keys, but could you please be kind and set a good example? 🙂

```suggestion
      # -- Create RBAC for admission webhook patch job or not
      create: true
```

Do not forget to generate the docs afterwards. Thank you!

---

@Gacko Thanks for the suggestions.
I have implemented options for `ServiceAccount`.
Added some tests (not an expert on these feel free to suggest if its not right).